### PR TITLE
Fix protocols in team export service [SCI-3037]

### PR DESCRIPTION
**Pull request by okriuchykhin**
Jira ticket: [SCI-3037](https://biosistemika.atlassian.net/browse/SCI-3037)

### What was done
Refactored and fixed team export service

### CODE REVIEW

File: app/services/model_exporters/team_exporter.rb [hunk 1]
```diff
@@ -57,7 +54,9 @@ def team(team)
         custom_fields: team.custom_fields,
         repositories: team.repositories.map { |r| repository(r) },
         tiny_mce_assets: team.tiny_mce_assets,
-        protocols: team.protocols.where(my_module: nil).map { |pr| protocol(pr) },
+        protocols: team.protocols.where(my_module: nil).map do |pr|
+          protocol(pr)
+        end,
```
Comment (mlorb) [inline app/services/model_exporters/team_exporter.rb]: I think this change is not needed. One liner is better.

File: app/services/model_exporters/experiment_exporter.rb [hunk 2]
```diff
@@ -3,10 +3,8 @@
 module ModelExporters
   class ExperimentExporter < ModelExporter
     def initialize(experiment_id)
-      @experiment = Experiment.find_by_id(experiment_id)
-      raise StandardError, 'Can not load experiment' unless @experiment
```
Comment (mlorb) [inline app/services/model_exporters/experiment_exporter.rb]: We don't need to handle error anymore?

File: app/services/model_exporters/team_exporter.rb [hunk 3]
```diff
@@ -3,11 +3,8 @@
 module ModelExporters
   class TeamExporter < ModelExporter
     def initialize(team_id)
-      @team = Team.includes(:user_teams).find_by_id(team_id)
-      raise StandardError, 'Can not load team' unless @team
```
Comment (mlorb) [inline app/services/model_exporters/team_exporter.rb]: same here

File: app/services/model_exporters/experiment_exporter.rb (hunk 2 above)
Comment (okriuchykhin) [reply app/services/model_exporters/experiment_exporter.rb]: .find() throws an exception if object not found

File: app/services/model_exporters/team_exporter.rb (hunk 3 above)
Comment (okriuchykhin) [reply app/services/model_exporters/team_exporter.rb]: .find() throws an exception if object not found

File: app/services/model_exporters/team_exporter.rb (hunk 1 above)
Comment (okriuchykhin) [reply app/services/model_exporters/team_exporter.rb]: than it will not fit into the line

**Pull request merged**